howto/gpu/increase-instance-density.md

```diff
@@ -1,7 +1,7 @@
 (howto-increase-instance-density)=
 # Increase instance density
 
-In an Anbox Cloud deployment with NVIDIA GPUs, launching multiple GPU-based Anbox instances or performing {ref}`howto-run-benchmarks` may reveal a limit on the number of running instances. Although [nvidia-smi](https://developer.nvidia.com/system-management-interface) shows low GPU memory usage and utilization, launching additional instances can result in the following errors when [viewing instance logs](/howto/troubleshoot/view-logs.md):
+In an Anbox Cloud deployment with NVIDIA GPUs, launching multiple GPU-based Anbox instances or performing {ref}`howto-run-benchmarks` may reveal a limit on the number of running instances. Although [NVIDIA-SMI](https://developer.nvidia.com/system-management-interface) shows low GPU memory usage and utilization, launching additional instances can result in the following errors when [viewing instance logs](/howto/troubleshoot/view-logs.md):
```

Comment:
Upstream seems to call it 'NVIDIA-smi' and 'nvidia-smi' if you look at https://developer.nvidia.com/system-management-interface

I'm not sure what we should call it here then.